airflow/www/templates/airflow/dags.html

```diff
@@ -28,6 +28,17 @@
 {% block content %}
   <h2>DAGs</h2>
 
+{% if not is_user_logged_in %}
+<div class="alert alert-warning" role="alert">
+  Note: Some DAGs may not be visible until you <a href="/login/">log in</a>.
+</div>
+{% elif is_public_user %}
+<div class="alert alert-warning" role="alert">
+  Note: Some DAGs may not be visible because you do not have sufficient privileges.
```

Comment:
This warning could equally apply to every other role except Admin.

How about only showing this if `len(dags) == 0`

Reply:
Sure, I can change that